react/components/molecules/search/search.js

```diff
@@ -0,0 +1,148 @@
+import React, { PureComponent } from "react";

```

Comment:
Merge with master to get rid of this from the diff

Reply:
search is not in master but it was already accepted and merged here